### PR TITLE
[13.0][IMP] stock_landed_costs_security: Restrict top level menu permission

### PR DESCRIPTION
**Pull request by cesar-tecnativa**
@Tecnativa TT37364

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please change the commit message to follow the proper structure (like the one I have put in the PR title).

File: stock_landed_costs_security/views/stock_landed_costs_view.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
         id="menu_landed_costs"
         sequence="30"
         action="stock_landed_costs.action_stock_landed_cost"
-        groups="stock.group_stock_manager,stock.group_stock_user,stock_landed_costs_security.group_stock_landed_costs_security_user"
+        groups="stock_landed_costs_security.group_stock_landed_costs_security_user"
```
Comment (pedrobaeza) [inline stock_landed_costs_security/views/stock_landed_costs_view.xml:22]:
```suggestion
        groups="group_stock_landed_costs_security_user"
```

**Review by pedrobaeza — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Sorry, I told you to put relative XMl-ID, but it seems it should be fully qualified. Please change it back again.

**Pull request merged**